setup/install.php

```diff
@@ -171,6 +176,8 @@
                 define('DB_PASSWD', $_POST['dbpw']);
                 define('DB_NAME', $_POST['dbname']);
                 define('DB_PREFIX', $_POST['dbprefix']);
+                $dbSsl = $_POST['dbSsl'];
+                define('DB_SSL', $dbSsl == "1" ? true : false);
```

Comment:
```suggestion
                define('DB_SSL', $dbSsl == "1" ? 1 : 0);
```

常量的值需要前后一致，要不统一为 true/false，要不就 1/0

Reply:
已更正为统一用1/0